### PR TITLE
Adjust evo strings for HGSS dex

### PR DESCRIPTION
**Pull request by Pawkkie**
## Description
- Changes LvlUp text to use {LV}{UP_ARROW} instead
- Fixes Sylveon's evolution text
- Change "mega evolves" to "Mega Evolves", thanks to Jasper/Bassoonian for catching :)

## Images
Before:
Before

After:
After

## **Discord contact info**
@Pawkkie

### CODE REVIEW

File: src/strings.c [hunk 1]
```diff
@@ -1911,47 +1911,48 @@ const u8 gText_EVO_Buttons_Decapped_PE[] = _("{DPAD_UPDOWN}Evos  {A_BUTTON}Check
...
+const u8 gText_EVO_BEAUTY[] = _("{LV}{UP_ARROW}, high beauty");
+const u8 gText_EVO_LEVEL_FEMALE[] = _("{LV}{UP_ARROW} to {STR_VAR_2}, is female");
+const u8 gText_EVO_LEVEL_MALE[] = _("{LV}{UP_ARROW} to {STR_VAR_2}, is male");
+const u8 gText_EVO_LEVEL_NIGHT[] = _("{LV}{UP_ARROW} to {STR_VAR_2}, night");
+const u8 gText_EVO_LEVEL_DAY[] = _("{LV}{UP_ARROW} to {STR_VAR_2}, day");
+const u8 gText_EVO_LEVEL_DUSK[] = _("{LV}{UP_ARROW} to {STR_VAR_2}, dusk (5-6PM)");
+const u8 gText_EVO_ITEM_HOLD_DAY[] = _("{LV}{UP_ARROW}, holds {STR_VAR_2}, day");
+const u8 gText_EVO_ITEM_HOLD_NIGHT[] = _("{LV}{UP_ARROW}, holds {STR_VAR_2}, night");
+const u8 gText_EVO_MOVE[] = _("{LV}{UP_ARROW}, knows {STR_VAR_2}");
+const u8 gText_EVO_MOVE_TYPE[] = _("{LV}{UP_ARROW}, knows {STR_VAR_2} move");
```
Comment (AsparagusEduardo) [inline src/strings.c:1938]: You didn't remove `gText_EVO_MOVE_TYPE`

**Review by AsparagusEduardo — CHANGES_REQUESTED** [review]

File: src/strings.c (hunk 1 above)
Comment (Pawkkie) [reply src/strings.c]: Oh heck my bad, just pushed a correction

File: include/strings.h
Comment (AsparagusEduardo) [inline include/strings.h:1]: You haven't deleted `gText_EVO_MOVE_TYPE` from this file.

Comment (Pawkkie) [reply include/strings.h:1]: I promise I do have functioning eyes :D

**Review by AsparagusEduardo — APPROVED** [review]

**Pull request merged**